### ISSUE TITLE
[BUG] Cannot place a minion on the ground after placing one before

### ISSUE THREAD

**Issue by mnhatt27** [Artillex-Studios/AxMinions#37]
Description of the bug
The minions were fine until now, my configuration didn't have a limit on the number of minions placed on the ground, but one day it crashed and I couldn't use them, for example, if I placed one before, it meant I couldn't use other minions, and I couldn't throw them away or anything.

Expected behaviour
instead it will function normally, can be placed on the ground many slaves

Steps to reproduce
[latest.log](https://github.com/user-attachments/files/17528727/latest.log)
Here is a video describing the error: https://streamable.com/t0gn7y

Which plugin is this issue with?
axminions

Versions
![image](https://github.com/user-attachments/assets/d4e70328-cc50-4e67-a7ec-fa1f5f513a91)

![image](https://github.com/user-attachments/assets/ea64699f-b70c-4278-bd2d-54fe1e199a82)

**Comment by drips0001** [comment]
Yeah, I am having same issue, I I place on minion then break it and after sometimes it will get kind of useless you can't throw minion head (but you can place in chest to get rid of it) you can't place it, you can't do anything.

**Comment by Balhorn** [comment]
Same issue here

**Comment by Balhorn** [comment]
My friend picked it up and put it in the chest, when we restarted the server minion is back in the spot from where we picked it and there is a minion in the chest so it basically duped but maybe this information can help you locate the issue

**Comment by drips0001** [comment]
It's related to databse as far as I think so,
Couse there is a setting called auto-save-minutes and default it set to 3, this is the exact time when minions start behaving abnormally like once picked up then will not place, will not get out of inventory, etc...
